pypgatk/commands/threeframe_translation.py

```diff
@@ -10,10 +10,9 @@
 
 
 @click.command('threeframe-translation', short_help="Command to perform 3'frame translation")
-@click.option('--config_file',
-              '-c',
+@click.option('-c', '--config_file',
               help='Configuration to perform conversion between ENSEMBL Files',
-              default= this_dir + '../config/ensembl_config.yaml')
+              default= this_dir + '/../config/ensembl_config.yaml')
```

Comment:
this_dir introduces error when the default is used.

Reply:
how are you running it, for me is working from the package.